File: DuDoong-Domain/src/main/java/band/gosrock/domain/domains/event/exception/EventCannotEndBeforeStartException.java
package band.gosrock.domain.domains.event.exception;


import band.gosrock.common.exception.DuDoongCodeException;

public class EventCannotEndBeforeStartException extends DuDoongCodeException {

    public static final DuDoongCodeException EXCEPTION = new EventCannotEndBeforeStartException();

    private EventCannotEndBeforeStartException() {
        super(EventErrorCode.EVENT_CANNOT_END_BEFORE_START);
    }
}

File: DuDoong-Domain/src/main/kotlin/band/gosrock/domain/domains/event/exception/EventCannotEndBeforeStartException.kt
package band.gosrock.domain.domains.event.exception

import band.gosrock.common.exception.DuDoongCodeException

class EventCannotEndBeforeStartException private constructor() : DuDoongCodeException(EventErrorCode.EVENT_CANNOT_END_BEFORE_START) {
    companion object {
        @JvmField
        val EXCEPTION: DuDoongCodeException = EventCannotEndBeforeStartException()
    }
}
